CardData.py

```diff
@@ -0,0 +1,330 @@
+#!/usr/bin/python3
+import re
+import GwentUtils
+
+IMAGE_SIZES = ['original', 'high', 'medium', 'low', 'thumbnail']
+
+CRAFT_VALUES = {}
+CRAFT_VALUES['Common'] = {"standard": 30, "premium": 200, "upgrade": 100}
+CRAFT_VALUES['Rare'] = {"standard": 80, "premium": 400, "upgrade": 200}
+CRAFT_VALUES['Epic'] = {"standard": 200, "premium": 800, "upgrade": 300}
+CRAFT_VALUES['Legendary'] = {"standard": 800, "premium": 1600, "upgrade": 400}
+
+MILL_VALUES = {}
+MILL_VALUES['Common'] = {"standard": 10, "premium": 10, "upgrade": 20}
+MILL_VALUES['Rare'] = {"standard": 20, "premium": 20, "upgrade": 50}
+MILL_VALUES['Epic'] = {"standard": 50, "premium": 50, "upgrade": 80}
+MILL_VALUES['Legendary'] = {"standard": 200, "premium": 200, "upgrade": 120}
+
+CATEGORIES = {
+    "Aedirn": "Aedirn",
+    "Alchemy": "Alchemy",
+    "Ambush": "Ambush",
+    "An_Craite": "An Craite",
+    "Banish_In_Graveyard": "Doomed",
+    "Bear": "Bear",
+    "Beast": "Beast",
+    "Blitz": "Blitz",
+    "Blue_Stripes": "Blue Stripes",
+    "Breedable": "Breedable",
+    "Brokvar": "Brokvar",
+    "Cintra": "Cintra",
+    "Construct": "Construct",
+    "Cursed_One": "Cursed",
+    "Devourer": "Devourer",
+    "Dimun": "Dimun",
+    "Double_Agent": "Double Agent",
+    "Draconid": "Draconid",
+    "Dragon": "Dragon",
+    "Drummond": "Drummond",
+    "Dwarf": "Dwarf",
+    "Dyrad": "Dryad",
+    "Elf": "Elf",
+    "Harpy": "Harpy",
+    "Heymaey": "Haymaey",
+    "Insectoid": "Insectoid",
+    "Kaedwen": "Kaedwen",
+    "Leader": "Leader",
+    "Mage": "Mage",
+    "Medic": "Medic",
+    "Necrophage": "Necrophage",
+    "Non_Decoyable": "Stubborn",
+    "Non_Medicable": "Permadeath",
+    "Officer": "Officer",
+    "Ogroid": "Ogroid",
+    "Organic": "Organic",
+    "Potion": "Potion",
+    "Redania": "Redania",
+    "Regressing": "Regressing",
+    "Relict": "Relict",
+    "Shapeshifter": "Shapeshifter",
+    "Soldier": "Soldier",
+    "Special": "Special",
+    "Specter": "Specter",
+    "Spell": "Spell",
+    "Spy": "Agent",
+    "Support": "Support",
+    "Svalblod": "Svalblod",
+    "Tactic": "Tactic",
+    "Temeria": "Temeria",
+    "Tordarroch": "Tordarroch",
+    "Tuirseach": "Tuirseach",
+    "Vampire": "Vampire",
+    "Vodyanoi": "Vodyanoi",
+    "War_Machine": "Machine",
+    "Weather": "Weather",
+    "Wild_Hunt": "Wild Hunt",
+    "Witcher": "Witcher"
+}
+
+class CardData:
+    def __init__(self, gwentDataHelper):
+        self._helper = gwentDataHelper
```

Comment:
`gwentDataHelper` need not be assigned to `self` as it is only used during constructor initialization